src/containers/layouts/auth/signUp2/signUp2.component.tsx

```diff
@@ -59,6 +56,10 @@ class SignUp2Component extends React.Component<SignUp2Props, State> {
     this.props.onSignUpPress(this.state.formValue);
   };
 
+  private keyboardOffset = (height: number): number => {
+    return height - 56;
```

Comment:
Can it be calculated for the any of use-cases?

Reply:
> Currently there is no way to handle it, if not only using onLayout (and wrapping bottom components into one container in some cases)